### PR TITLE
Push otp jar file to S3 bucket

### PR DESCRIPTION
**Pull request by AdamWiktor**

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -41,3 +41,10 @@ jobs:
         run: |
           docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+      - name: Copy otp jar artifact to S3 bucket
+        env:
+          JAR_FILE: otp_${{ github.sha }}.jar
+          BUCKET_NAME: s3://${{ S3_BUCKET_NAME }}
```
Comment (Kamyki) [inline .github/workflows/push.yml]: Where those variables are set? In some github control panel or sth?

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -41,3 +41,10 @@ jobs:
...
+        run: |
+          cp ./target/otp-1.5.0-SNAPSHOT-shaded.jar $JAR_FILE
```
Comment (Kamyki) [inline .github/workflows/push.yml]: Is it for changing file name? Maybe following will work?
```
aws s3 cp ./target/otp-1.5.0-SNAPSHOT-shaded.jar $BUCKET_NAME/$JAR_FILE
```

**Review by Kamyki — APPROVED** [review]

File: .github/workflows/push.yml (hunk 1 above)
Comment (AdamWiktor) [reply .github/workflows/push.yml]: This is added as a repository secret, there should be `secrets.S3_BUCKET_NAME`. Fixed.

File: .github/workflows/push.yml (hunk 2 above)
Comment (AdamWiktor) [reply .github/workflows/push.yml]: I just checked it in documentation, and it is possible to do it like suggested. Fixed.

**Pull request merged**